### DISCUSSION TITLE
[1123770]Curse of the Dead Gods 无间冥寺 本体

### DISCUSSION THREAD

**Discussion by lishen2004328** [pjy612/SteamManifestCache#245] [Share]
`appid:1123770`
Key:
```
1123771;1a98ebf1f0691fe81c354b0a0d89224f4652aa3696853b7040d5492e272bb48a
```
email:[1148997557@qq.com](mailto:1148997557@qq.com)
code:F13BB79DEADD443A

**Comment by pjy612** [comment]
密钥不正确,格式也不对
https://steamdb.info/app/1123770/depots/

**Reply by lishen2004328** [reply]
"1123771"
					{
						"DecryptionKey"		"1a98ebf1f0691fe81c354b0a0d89224f4652aa3696853b7040d5492e272bb48a"
					}
					"1123770"
					{
						"DecryptionKey"		"d0d53200d054f56b72381e9e109bcb806aadeb5d8c24ec2b976f5e9958686d83"
					1123770是本体 3771应该是dlc

**Reply by lishen2004328** [reply]
我改了一下 应该好了

**Reply by pjy612** [reply]
你看看别的帖子里的格式，顶置模板也写清楚了

**Reply by lishen2004328** [reply]
已修改

**Reply by pjy612** [reply]
查收
